client/src/components/new-request/stats.vue

```diff
@@ -1,16 +1,17 @@
 <template>
   <v-container class="mt-1 mb-n3 py-0" fluid>
     <v-row class="float-right stats-v-row">
-        <div class="stats-content-outer py-0">
-          <div class="stats-content-inner-1 text-center">
-            <div class="stats-value h3-lt">{{ autoApprovedCount }}</div>
-            <div class="stats-unit">Names</div>
-          </div>
-          <div class="stats-content-inner-2">
-            <u>Auto-Approved</u> Today.<br>
-            No Wait Time
-          </div>
-        </div>
+<!--      COMMENTED OUT FOR FUTURE IMPLEMENTATION-->
```

Comment:
Are you using VS Code? What do you get if you select this block and press CTRL+/ ?

Reply:
I use jetbrains (webstorm) in this case.
I don't get anything actually haha. My keybind is WIN + / to commented and uncomment blocks.

Reply:
I was surprised to see every line as its own comment. VS Code just adds a start and end tag for the whole block.

Reply:
ohhhh i see. Yea, i think i could do that as well. Maybe that is a preference i could change in the editor.